client-src/elements/chromedash-typeahead.ts

```diff
@@ -91,6 +91,16 @@ export class ChromedashTypeahead extends LitElement {
     this.slDropdownRef.value!.show();
   }
 
+  focus() {
+    const slInput: SlInput = this.slInputRef.value as SlInput;
+    slInput?.focus();
+  }
+
+  blur() {
```

Comment:
Is this used?

Reply:
No, I just felt that if I added focus(), it should also have blur().